docs/queries/layout.md

```diff
@@ -28,6 +28,7 @@ The following elements exist:
 - `backlink`
 - `urgency`
 - `priority`
+- `created date`
```

Comment:
thank you for adding this.

You don't need to to this if you don't feel like it, but before this feature is released, the files called something like 'quick-reference/index.md' and 'getting-started/dates.md' will also need to be updated.

And where that 'dates.md' file is updated, it would be really helpful for it to list the things that do not yet work with created date (like, you can't search, sort and group) - to reduce the number of people creating feature requests for them.

Reply:
Update index.md and dates.md files with new functionality. Don't know if wording is correct, but section has been added. :)